tests/test_conditional_steps.py

```diff
@@ -0,0 +1,57 @@
+import re
+import pytest
+
+from universum import __main__
+
+
+true_branch_step_name = "true branch"
+false_branch_step_name = "false branch"
+
+
+def test_conditional_true_branch(tmpdir, capsys):
+    check_conditional_step_success(tmpdir, capsys, conditional_step_passed=True)
+
+
+def test_conditional_false_branch(tmpdir, capsys):
+    check_conditional_step_success(tmpdir, capsys, conditional_step_passed=False)
+
+
+def check_conditional_step_success(tmpdir, capsys, conditional_step_passed):
+    config_file = build_config_file(tmpdir, conditional_step_passed)
+    check_conditional_step(tmpdir, capsys, config_file, conditional_step_passed)
+
+
+def build_config_file(tmpdir, conditional_step_passed):
+    conditional_step_exit_code = 0 if conditional_step_passed else 1
+    config = "from universum.configuration_support import Configuration, Step\n"
+    config += f"true_branch_step = Step(name='{true_branch_step_name}', command=['pwd'])\n"
```

Comment:
Please use triple quotes instead of line adding for better readability

Reply:
For even better readability I suggest to use `inspect.cleandoc`